### PR TITLE
Jws_tut a st bud

### PR DESCRIPTION
**Pull request by JiaweiSun0512**

### CODE REVIEW

File: tutorials/pipelines/tut_a_st_bud/tut_a_st_bud.tex [hunk 1]
```diff
@@ -2,15 +2,246 @@
 \usepackage{../braph2_tut}
 %\geometry{showframe} % display margins for debugging page layout
 
-\title{Pipeline for Comparison of Structural Data using Binary Undirected graphs at fixed Densities}
+\title{Pipeline for Comparison of Banarized Structural Data between Groups of Subjects at Fixed Densities}
```
Comment (annacanal) [inline tutorials/pipelines/tut_a_st_bud/tut_a_st_bud.tex]: **Binarized**

File: tutorials/pipelines/tut_a_st_bud/tut_a_st_bud.tex [hunk 2]
```diff
@@ -2,15 +2,246 @@
...
 
 \begin{document}
 
 \maketitle
 
 \begin{abstract}
 \noindent
-This tutorial is not ready. You can find a similar one in Structural WU (\url{https://github.com/softmatterlab/BRAPH-2-Matlab/blob/develop/tutorials/pipelines/tut_a_st_wu/tut_a_st_wu.pdf}).
+In this tutorial, we will upload a file containing the pipeline with the different steps to compare two groups of subjects using \emph{structural data} (check tutorial \emph{Group of Subjects with Structural Data}) and banarized undirected graphs. This Tutorial explains how to perform group analyses and comparisons with this kind of data at fixed densities.
```
Comment (annacanal) [inline tutorials/pipelines/tut_a_st_bud/tut_a_st_bud.tex]: **Binarized**

**Review by annacanal — COMMENTED** [review]
Overall looks good. Please review the misspellings such as banarized => **Binarized**

File: tutorials/pipelines/tut_a_st_bud/readme.md
Comment (bzufiriaki) [inline tutorials/pipelines/tut_a_st_bud/readme.md:1]: Include this readme file again and put the following:

# BRAPH 2.0 — Pipeline for Comparison of Structural Data using Binary Undirected graphs at fixed Densities

This tutorial is only available as a pdf at the moment (../tut_a_st_bud/tut_a_st_bud.pdf).

**Review by bzufiriaki — CHANGES_REQUESTED** [review]

**Pull request merged**